picard/track.py

```diff
@@ -368,6 +375,23 @@ def _convert_folksonomy_tags_to_genre(self):
             join_with=config.setting['join_genres']
         )
 
+    def _add_tags(self, tags, name):
+        self.metadata[name] = tags.keys()
+
+    def _add_folksonomy_tags(self):
+        tags = Counter(self._folksonomy_tags)
+        tags += self.album._folksonomy_tags
+        self._add_tags(tags, '_folksonomy_tags')
+
+    def _add_genres(self):
+        genre = Counter(self._genres)
+        genre += self.album._genres
+        self._add_tags(genre, '_genres')
+
+    def _delete_genres_from_tags(self):
```

Comment:
Here we're in `Track` and delete stuff in `Album`, what about moving this method to `Album` instead and calling it from here?

Reply:
Fixed in: 539e104fb2deec4b10c87359b09020ba51b88e84